packages/design-system/src/components/InlineEditing/Primitives/InlineEditingPrimitive.tsx

```diff
@@ -1,5 +1,11 @@
+import { ButtonIcon } from '../../ButtonIcon';
```

Comment:
Hum, is it wanted to change the order of imports ? Looks like the prettier plugin was not running 🤔

Reply:
Import order looks really weird indeed

Reply:
problem is from prettierrc file i think. It doesn't import generic one and has no rule about import order. I will update prettierrc file